less/components/favorite.less

```diff
@@ -32,7 +32,11 @@
+  }
```

Comment:
I am generating the Base64 more or less by hand from the SVG's from James. And you're right, Less has `data-uri` which will do the embed for us. Ideally we'd just let the browser load the SVG asset directly but I seem to recall running into a problem with that while developing the component.